### ISSUE TITLE
Remove Chromium (Headless) Support

### ISSUE THREAD

**Issue by brusch** [pimcore/pimcore#16631]
### Improvement description

Functionality is now covered by Gotenberg, no need to maintain 2 implementations. 

See also: #16608

**Comment by storbahn** [comment]
Is it possible to use Gotenberg without docker?

**Comment by dkarlovi** [comment]
@storbahn yes, Gotenberg is just using a remote Chrome, but the idea is basically the same as before. You can provide the correctly provisioned remote Chrome instance any way you want.
